### ISSUE TITLE
Missing addressbook share icon in settings pane, when using ad-blocking addons

### ISSUE THREAD

**Issue by irgendwie** [owncloud/contacts#508]
Some users (ref. https://help.nextcloud.com/t/chrome-problem-with-sharing-contacts/1834) experience problems with missing share icons for addressbooks if using ad-blocking addons like ublock or adblock.
The browser doesn't seem to matter, since issues in Firefox and Chrome are reported.

Since I **don't** experience this issue, it would be very helpful if someone could dig into this issue and find the static filter list, which is causing this issue.

When using [uBlock Origin](https://github.com/gorhill/uBlock), one could take a look into the [logger](https://github.com/gorhill/uBlock/wiki/The-logger) to find the "culprit".
## TLDR, workaround:
- Disable the ad-blocking addon and see if it works

**Comment by irgendwie** [comment]
Occurs when using:
- Fanboy’s Annoyance List (Static filter `##.icon-share`)
- Fanboy’s Social Blocking List (Static filter `##.icon-share`)
- ...

https://github.com/owncloud/contacts/blob/master/templates/addressBook.html#L19

**Comment by irgendwie** [comment]
This issue is not a problem of the contacts app, but rather a problem of the filter lists for using too generic filters. Won't fix for now.

## TLDR, workaround:
- Disable your ad-blocking addon for your cloud

**Comment by smed79** [comment]
> Disable your ad-blocking addon for your cloud

Not need to disable **[ad-blocking addon](https://adblockplus.org/)**, you have a choice between:
- disable [Fanboy's Social Blocking list](https://adblockplus.org/en/features#socialmedia) (or [Fanboy's Annoyances list](https://adblockplus.org/subscriptions#type_other)) **or**
- add a filter like this `#@#.icon-share` (see [How do I add a customer filter?](https://adblockplus.org/forum/viewtopic.php?f=2&t=42013))

**Comment by ryanbr** [comment]
Given we can't whitelist all the domains, if owncloud changed to a less social element name that'd be far easier.
